pkg/runner/runner.go

```diff
@@ -5,11 +5,11 @@ import (
 	"bytes"
 	"fmt"
 	"io"
-	"strings"
 
 	"code.google.com/p/go-uuid/uuid"
 
 	"github.com/fsouza/go-dockerclient"
+	"github.com/mattn/go-shellwords"
```

Comment:
Can we import this as `shellwords "github.com/mattn/go-shellwords"`. Without it, goimports will remove it if you haven't `godep restore`'d.